src/store/getters/state-getters.ts

```diff
@@ -1,177 +1,197 @@
-// Enums
 import { AccountTypes, CorpTypeCd, RoleTypes } from '@/enums'
```

Comment:
Where I went with this is:
- all store getters should be `isXXX` or `getXXX`
- all store setters should be `setXXX`
- return a specific value (eg, `state.stateModel.myObject.myProperty`) if it makes sense (ie, used a lot), otherwise try to return `myObject` (which assumes the component that use myObject know what's in it, which I think is fine) (but for a component to know about the structure of the store is not fine IMHO)
- have getters call other getters where feasible, which will make refactoring the store objects (if needed) easier